### ISSUE TITLE
Problem: no FAQ page

### ISSUE THREAD

**Issue by gazhayes** [Blockrazor/blockrazor#1014]
Problem: when people start looking at the site there are a few common questions they are going to ask. 

solution: add a FAQ page. Put a font-awesome help icon in the navbar to link to it.

I'll write the content if someone else does the routes etc.

**Comment by AshHimself** [comment]
I can do this before I fall to sleep :)

**Comment by samuelralak** [comment]
I don't support the idea of adding this to the navbar as IMO it would clatter the top navbar. I would suggest adding this link (together with the About, T&C e.t.c) on the side menu, possibly at the very bottom.

**Comment by AshHimself** [comment]
I agree its getting a wee bit cluttered. I did add it to topbar now but can change if need be.

Could use this instead if you prefer. https://getbootstrap.com/docs/4.0/components/collapse/#accordion-example

Also to make more room the br logo text should disappear or only show the small logo (when it exists).

**Comment by gazhayes** [comment]
It needs to be very obvious though. The 'share' button can be moved elsewhere (could actually add this on the bounty page as it's basically a type of bounty and that's where people will be going to earn KZR).

**Comment by AshHimself** [comment]
@gazhayes  updated in #1048
